nheko.appdata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.mujx.Nheko.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPLv3+ and Public Domain and MIT and Boost and ASL 2.0 and CC-BY</project_license>
```

Comment:
I'm not sure if it's correct, probably should be just GPLv3+ and CC-BY (for assets?)

Reply:
Most of these aren't valid license names anyway: https://spdx.org/licenses/